[bug][v2] build error in aarch64 windowsOS

### Describe the bug

Build failure on the WindowsOS of the aarch64 chip. 

Probably a problem that appeared after alpha.21.


```bash
error[E0412]: cannot find type `size_t` in the crate root
   --> C:\Users\dongruilin\.cargo\registry\src\index.crates.io-6f17d22bba15001f\winapi-0.2.8\src\ws2def.rs:274:19
    |
274 |     ai_bloblen: ::size_t,
    |                   ^^^^^^ help: a type alias with a similar name exists: `SIZE_T`
    |
   ::: C:\Users\dongruilin\.cargo\registry\src\index.crates.io-6f17d22bba15001f\winapi-0.2.8\src\basetsd.rs:86:1
    |
86  | pub type SIZE_T = ULONG_PTR;
    | ---------------------------- similarly named type alias `SIZE_T` defined here
    |
note: found an item that was configured out
   --> C:\Users\dongruilin\.cargo\registry\src\index.crates.io-6f17d22bba15001f\winapi-0.2.8\src\lib.rs:366:10
    |
366 | pub type size_t = c_uint;
    |          ^^^^^^
    = note: the item is gated behind the `x86` feature
note: found an item that was configured out
   --> C:\Users\dongruilin\.cargo\registry\src\index.crates.io-6f17d22bba15001f\winapi-0.2.8\src\lib.rs:368:10
    |
368 | pub type size_t = __uint64;
    |          ^^^^^^
    = note: the item is gated behind the `x86_64` feature

   Compiling rustc_version v0.4.0
For more information about this error, try `rustc --explain E0412`.
error: could not compile `winapi` (lib) due to 285 previous errors
warning: build failed, waiting for other jobs to finish...
       Error failed to build app: failed to build app
```



### Reproduction

_No response_

### Expected behavior

_No response_

### Full `tauri info` output

```text
[✔] Environment
    - OS: Windows 10.0.22621 X64
    ✔ WebView2: 122.0.2365.52
    ✔ MSVC: Visual Studio Community 2022
    ✔ rustc: 1.76.0 (07dca489a 2024-02-04)
    ✔ cargo: 1.76.0 (c84b36747 2024-01-18)
    ✔ rustup: 1.26.0 (5af9b9484 2023-04-05)
    ✔ Rust toolchain: stable-aarch64-pc-windows-msvc (default)
    - node: 20.8.0
    - pnpm: 8.10.0
    - yarn: 4.0.2
    - npm: 10.1.0

[-] Packages
    - tauri [RUST]: 2.0.0-beta.7
    - tauri-build [RUST]: 2.0.0-beta.5
    - wry [RUST]: 0.37.0
    - tao [RUST]: 0.26.0
    - @tauri-apps/api : not installed!
    - @tauri-apps/cli [NPM]: 2.0.0-beta.1

[-] App
    - build-type: bundle
    - CSP: unset
    - frontendDist: ../dist
    - devUrl: http://localhost:3000/
    - framework: React
    - bundler: Rollup
```


### Stack trace

_No response_

### Additional context

_No response_

--- a/core/tauri-plugin/Cargo.toml
+++ b/core/tauri-plugin/Cargo.toml
@@ -35,7 +35,7 @@ serde_json = { version = "1", optional = true }
 glob = { version = "0.3", optional = true }
 toml = { version = "0.8", optional = true }
 schemars = { version = "0.8", features = [ "preserve_order" ] }
-walkdir = { version = "1", optional = true }
+walkdir = { version = "2", optional = true }
 
 [target."cfg(target_os = \"macos\")".dependencies]
 plist = { version = "1", optional = true }

